api/r/triple.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: default
+title: Triple
+nav_order: 2
+parent: R
+grand_parent: API
+---
+
+# Show Triples Associated with Node(s)
+
+## `get_triples(dcids, limit=MAX_LIMIT)`
+
+Given a list of nodes, return triples which are associated with the specified
+node(s).
+
+A knowledge graph can be described as a collection of *triples* which are
+3-tuples that take the form *(s, p, o)*. Here, *s* and *o* are nodes in the
+graph called the *subject* and *object* respectively, while *p* is the property
+label of a directed edge from *s* to *o* (sometimes also called the *predicate*).
+
+**Arguments**
+
+* `dcids` (`vector` of strings) - DCIDs to get triples for.
+
+* `limit` (`numeric`, optional) - The maximum number of triples per combination of
+    property and type associated with nodes linked by that property to fetch,
+    ≤ 500.
+
+**Returns**
+
+A *named* `vector` mapping DCIDs to a `vector` of triples *(s, p, o)* where *s*, *p*, and *o* are
+strings and either the *subject* or *object* is the specified DCID.
+
+**Raises**
+
+* `Response error` - If the payload returned by the Data Commons REST API is malformed or the API key is not set.
+
+Be sure to initialize the library, and specify the API key. Check the [R library setup guide](/api/r/) for more details.
+
+## Examples
+
+We would like to get one triple per property and type combination associated with
+[California](https://browser.datacommons.org/kg?dcid=geoId/06):
+
+```r
+> library(datacommons)
+> set_api_key(YOUR_API_KEY_HERE)
+> get_triples(c('geoId/06'), limit=1)
+$`geoId/06`
```

Comment:
isn't there a more succinct way to display these results? :/

Reply:
This is R's output for the func, I can take a look at some postprocessing for the sake of nice docs.